code/modules/mob/living/emote.dm

```diff
@@ -201,6 +206,13 @@
 	key_third_person = "grimaces"
 	message = "grimaces."
 
+/datum/emote/living/growl
+	key = "growl"
+	key_third_person = "growls"
+	message = "growls."
+	message_mime = "looks angry!"
+	emote_type = EMOTE_AUDIBLE
+
```

Comment:
i'm gonna be real with you chief, if someone growled at me in real life, i'd think they're either a bit touched in the head, or a furry.